src/activity_handlers.cpp

```diff
@@ -2307,6 +2308,10 @@ void activity_handlers::oxytorch_finish( player_activity *act, player *p )
         here.ter_set( pos, t_pit );
         here.spawn_item( pos, itype_spike, rng( 1, 19 ) );
         here.spawn_item( pos, itype_scrap, rng( 1, 8 ) );
+    } else if( ter == t_retractable_gate_c || ter == t_retractable_gate_l ) {
```

Comment:
Open gate should be added too, like in sawing activity?

Reply:
I didn't include the open gate since it's retracted up into the ceiling.  You'd have to pull it closed in order to be able to reach the slats to cut through them.

Reply:
But you can saw it even if it's retracted up?

Reply:
Thanks! I missed that one when I changed the other.